cleanlab/internal/regression_utils.py

```diff
@@ -0,0 +1,28 @@
+"""
+Helper function internally used in cleanlab.regression
+"""
+
+import numpy as np
+
+
+def assert_valid_inputs(
+    labels: np.ndarray,
+    predictions: np.ndarray,
+    method: str,
+) -> None:
+    """Checks that ``labels``, ``pred_labels``, ``method`` are correctly formatted."""
+
+    # Check if labels and pred_labels are np.ndarray
+    if not isinstance(labels, np.ndarray) or not isinstance(predictions, np.ndarray):
+        raise TypeError("labels and pred_labels must be of type np.ndarray")
+
+    # Check if labels and pred_labels are of same shape
+    assert (
+        labels.shape == predictions.shape
+    ), f"shape of label {labels.shape} and predicted labels {predictions.shape} are not same."
```

Comment:
yes, I agree that there is some similarity between this block and `cleanlab.internal.validation.assert_valid_inputs`. Currently, this has been created to consider different parameters of regression as these problems may require some arguments that are entirely different from classification. Also, the nomenclature of other arguments might not make sense for the regression task for example `allow_one_class` has no valid significance in regression. 

It would be a good idea to move this to  `cleanlab.internal.validation` in the future.

Reply:
@krmayankb can you add todo in code?